### DISCUSSION TITLE
Add a Condition in Middleware

### DISCUSSION THREAD

**Discussion by PeranoXx** [MarJose123/filament-lockscreen#12] [Ideas]
add `$request->segment(1) == 'admin'` in Locker middleware. to apply locker only in admin panel and not it frontend

![image](https://user-images.githubusercontent.com/89066237/194837926-d7d59c64-88ba-4b12-8417-46ccb39bdb81.png)

**Comment by MarJose123** [comment]
> add `$request->segment(1) == 'admin'` in Locker middleware. to apply locker only in admin panel and not it frontend
> 
> ![image](https://user-images.githubusercontent.com/89066237/194837926-d7d59c64-88ba-4b12-8417-46ccb39bdb81.png)

I will think another approach to this as we cannot use this kind of approach as it will affect also other system who is using this plugin.

**Reply by PeranoXx** [reply]
yeah.. okay but make it for admin panel only.  😅😅

**Reply by MarJose123** [reply]
can you try this release on your testing https://github.com/MarJose123/filament-lockscreen/releases/tag/v1.1.0-beta

**Reply by MarJose123** [reply]
and update your config file for the lockscreen

https://github.com/MarJose123/filament-lockscreen/blob/4fafff873e478b9aa246b8c7ddf0f2ef7b89dbff/config/filament-lockscreen.php#L29-L33
